src/main/groovy/org/mockito/release/internal/gradle/ContinuousDeliveryPlugin.java

```diff
@@ -46,6 +46,19 @@ public void apply(final Project project) {
         project.getPlugins().apply(VersioningPlugin.class);
         project.getPlugins().apply(GitPlugin.class);
 
+        project.allprojects(new Action<Project>() {
+            @Override
+            public void execute(final Project project) {
+                project.getPlugins().withType(JavaLibraryPlugin.class, new Action<JavaLibraryPlugin>() {
+                    @Override
+                    public void execute(JavaLibraryPlugin plugin) {
+                        project.getTasks().getByName("publishToMavenLocal")
```

Comment:
I think "generatePomFileForJavaLibraryPublication" is the correct one. It is automagically generated from "generatePomFileFor" + publication name (specified in the BaseJavaLibraryPlugin) + "Publication". Fancy :)

I suggest to make a constant for this String inside BaseJavaLibraryPlugin because it will make the couplings between the plugins more clear.

Nice, you're learning how to hack & understand Gradle :)